pkg/auth/providers/saml/saml_provider.go

```diff
@@ -192,14 +220,18 @@ func (s *Provider) saveSamlConfig(config *v3.SamlConfig) error {
 	}
 
 	config.SpKey = common.GetName(config.Type, field)
+	if s.hasLdapGroupSearch() {
```

Comment:
same question here, `hasLdapGroupSearch` will return true for shibboleth, and `combineSamlAndLdapConfig` will return err if it fails to get ldapConfig, should `hasLdapGroupSearch` check for actual presence of ldapfields?